composer.json

```diff
@@ -33,11 +33,13 @@
         "contao/imagine-svg": "^0.2.2 || ^1.0",
         "dantleech/phpcr-migrations-bundle": "^1.0",
         "doctrine/annotations": "^1.2",
-        "doctrine/common": "^2.7.1",
```

Comment:
replaced by the subpackages doctrine/collections, doctrine/persistence, doctrine/cache which we use.